composer.json

```diff
@@ -39,7 +39,6 @@
     "laminas/laminas-dom": "^2.12",
     "phpunit/phpunit": "^7.5||^9.5",
     "yoast/phpunit-polyfills": "^1.1.0",
-    "doctrine/instantiator": "^1.5.0",
```

Comment:
We also need to update `composer.lock` for this change. Running `composer update` from with the repo dir should be enough.